torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3393,6 +3397,18 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard',
                                 device_type='cpu', dtypes=[torch.complex64])
                    )),
+    OpInfo('real',
```

Comment:
I'd suggest using `UnaryUfuncInfo` with corresponding numpy functions for reference.

That way we will get tests verifying the output of these function against numpy equivalents and multiple other tests (look at test_unary_ufuncs.py for reference)

You might have to tweak one or two functions there (you can refer the previous [PR](https://github.com/pytorch/pytorch/pull/51946/files) for some reference to the changes).